rules/windows/powershell/powershell_cmdline_specific_comb_methods.yml

```diff
@@ -0,0 +1,54 @@
+title: Encoded PowerShell Command Line
+id: cdf05894-89e7-4ead-b2b0-0a5f97a90f2f
+description: Detects specific combinations of encoding methods in the PowerShell command lines
+status: experimental
+references:
+    - https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse?slide=65
+tags:
+    - attack.defense_evasion
+    - attack.t1027
+    - attack.execution
+    - attack.t1059.001
+author: Teymur Kheirkhabarov (idea), Vasiliy Burov (rule), oscd.community
+date: 2020/10/11
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection1:
+        Image|endswith: '\powershell.exe'
+        CommandLine|all:
+            - '*char*'
+            - '*join*'
```

Comment:
you need to change `CommandLine|all:` to the `CommandLine|contains|all:` and remove `*` from the `char` and `join` (:

Reply:
Done. At first rule redaction I used this pipe, but during troubleshooting I simplified this construction :-)